_includes/tutorials/aggregating-count/flinksql/markup/dev/print-output-topic.adoc

```diff
@@ -0,0 +1,10 @@
+Seeing is believing, so let's query the persistent `movie_sales_by_year` table.
```

Comment:
This file can be deleted since it's not referenced anywhere. I left it in mistakenly in the `aggregating-minmax` tutorial. could you also delete `_includes/tutorials/aggregating-minmax/flinksql/markup/dev/print-output-topic.adoc` as part of this PR?

Reply:
yep will do